app/src/main/res/values/strings.xml

```diff
@@ -586,6 +586,7 @@
     <string name="folder">Folder</string>
     <string name="color_palette">Color Palette</string>
     <string name="app_intro_skip">Introduction Skipped!</string>
+    <string name="preference_BottomNavColor">getBottomNavBarColor</string>
```

Comment:
Pls follow the similar trend that is using power case letters in strong names.

Reply:
I don't understand you here. Please explain.